### ISSUE TITLE
unable to build on windows (WSL)

### ISSUE THREAD

**Issue by JoergWende** [ibm-messaging/mq-container#454]
Error building the image: 

# Build the new image
docker build --tag ibm-mqadvanced-server:9.2.2.0-amd64 --file Dockerfile-server --network build --build-arg MQ_URL=http://build:8080/IBM_MQ_9.2.2_LINUX_X86-64_NOINST.tar.gz --build-arg IMAGE_REVISION="4580cecf4973107dff184e8cbbcf9ac7f5b4e7df" --build-arg IMAGE_SOURCE="https://github.com/ibm-messaging/mq-container.git" --build-arg IMAGE_TAG="ibm-mqadvanced-server:9.2.2.0-amd64" --label version=9.2.2.0 --label name=ibm-mqadvanced-server --label build-date=2021-04-29T09:18:00+0200 --label architecture="amd64" --label run="docker run -d -e LICENSE=accept ibm-mqadvanced-server:9.2.2.0-amd64" --label vcs-ref=4580cecf4973107dff184e8cbbcf9ac7f5b4e7df --label vcs-type=git --label vcs-url=https://github.com/ibm-messaging/mq-container.git  --target mq-server .
Error response from daemon: network mode "build" not supported by buildkit
make: *** [Makefile:346: build-advancedserver] Error 1

**Comment by JoergWende** [comment]
solved by updating the docker json file with:

  "features": {
    "buildkit": false
  }

**Comment by ShashikanthRaoT** [comment]
Yep, I too faced the same issue but on Mac. Resolved with the same solution as you mention.
